Gemfile.lock

```diff
@@ -475,4 +478,4 @@ RUBY VERSION
    ruby 2.7.2p137
 
 BUNDLED WITH
-   2.2.10
+   2.2.15
```

Comment:
Can you bundle this with 2.2.10 ?

bundle _2.2.10_ install or perhaps revert to the old Gemfile.lock